core/src/main/java/com/linecorp/armeria/common/HttpStatus.java

```diff
@@ -377,14 +377,10 @@ public static HttpStatus valueOf(String statusText) {
 
     /**
      * Returns {@code true} if the content of the response for the specified status code is expected to
-     * be always empty (1xx, 204, 205 and 304 responses.)
+     * be always empty (204, 205 and 304 responses.)
      */
     @SuppressWarnings("checkstyle:OverloadMethodsDeclarationOrder")
     public static boolean isContentAlwaysEmpty(int statusCode) {
-        if (HttpStatusClass.INFORMATIONAL.contains(statusCode)) {
-            return true;
-        }
-
```

Comment:
Why was this removed?

Reply:
Because I think whether the content is empty or not is decided by the response headers, not the informational headers.
Informational headers will have the following response headers, so shouldn't we look at the response headers to make a decision if the following content will be empty or not?

Reply:
Doesn't it still mean that informational headers have no content?

Reply:
Had a quick chat with @minwoox and we chose to keep the modification in this PR, because we can't tell if the response will have no content from an informational status.